assignment1/Buynow.java

```diff
@@ -0,0 +1,180 @@
+package assignment1;
+
+import java.util.Scanner;
+
+public class Buynow {
+	public static void main(String[] args) {
+
+		Scanner scanner = new Scanner(System.in);
+		User newUser = new User();
+	 newUser.authenticate(scanner);
```

Comment:
User should be served a welcome message before performing any actions. let him get an idea what he is going to do in next couple of minutes